src/components/Filters/FilterByPrice/FilterByPrice.spec.js

```diff
@@ -0,0 +1,26 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import FilterByPrice from './FilterByPrice'
+
+describe('FilterByPrice', () => {
+  it('renders two input fields', () => {
+    render(<FilterByPrice />)
+    expect(screen.getByTestId('minPrice')).toBeInTheDocument()
+    expect(screen.getByTestId('maxPrice')).toBeInTheDocument()
+  })
+})
+
+/*   it('renders two inputs with the callback onChange', () => {
+    const callback = jest.fn()
+    render(
+      <FilterByPrice
+        onChange={callback}
+        setUserInputMaxPrice={callback}
+        setUserInputMinPrice={callback}
+      />
+    )
+    userEvent.type(screen.getByTestId('minPrice'), 0)
+    userEvent.type(screen.getByTestId('maxPrice'), 200)
+    expect(callback).toHaveBeenCalledTimes(2)
+    expect(callback).toHaveBeenCalledWith(0, 200)
+  })  */
```

Comment:
For this to work, I think you will need two callback functions. I think it could work like this, but not sure:
```suggestion
it('renders two inputs with the callback onChange', () => {
    const minPriceCallback = jest.fn()
    const maxPriceCallback = jest.fn()
    render(
      <FilterByPrice
        setUserInputMaxPrice={maxPriceCallback}
        setUserInputMinPrice={minPriceCallback}
      />
    )
    userEvent.type(screen.getByTestId('minPrice'), 0)
    userEvent.type(screen.getByTestId('maxPrice'), 200)
    expect(minPriceCallback).toHaveBeenCalled()
    expect(maxPriceCallback).toHaveBeenCalled()
    expect(minPriceCallback).toHaveBeenCalledWith(0)
    expect(maxPriceCallback).toHaveBeenCalledWith(200)
  })  
```